### ISSUE TITLE
Accessible buffer does not get new lines if buffer changes while focused

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#177882]
Saw this while testing https://github.com/microsoft/vscode/pull/177864:

Accessible buffer while focused and receiving output:

<img width="809" alt="image" src="https://user-images.githubusercontent.com/2193314/226727932-3b12b5fb-1832-4ff0-9e06-a5cc533b195d.png">

Going back to real buffer:

<img width="520" alt="image" src="https://user-images.githubusercontent.com/2193314/226727990-de77e5db-a7a1-44f5-9455-0169f44dbd03.png">

Back to accessible buffer:

<img width="814" alt="image" src="https://user-images.githubusercontent.com/2193314/226728004-ebf52854-d47c-4c87-af1e-358ea7b9c88b.png">

**Comment by meganrogge** [comment]
This used to work, so looks like a regression

**Comment by meganrogge** [comment]
easier to see what's going on with this repro
https://user-images.githubusercontent.com/29464607/226758368-17d2845c-4748-4c8e-97d3-9d39a0e25e27.mov

**Comment by meganrogge** [comment]
for each `writeParsed` call we are going from `startLine` to `end` but should just be getting `startLine` to prevent this overlap from happening
![Screenshot 2023-03-21 at 3 49 16 PM](https://user-images.githubusercontent.com/29464607/226759074-3cf328b0-6a12-482e-ad04-24ff57499e37.png)

**Comment by meganrogge** [comment]
This was fixed yesterday, but there are still issues - there shouldn't be missing or extra new lines like there are here


https://user-images.githubusercontent.com/29464607/226962260-b711b90b-d575-4ba0-ba72-591414089ac7.mov

**Comment by joyceerhl** [comment]
The buffer does get newlines now, but we do not scroll to the new lines. I assume that this is by design to reduce cognitive load for screen reader users, but if not please LMK and I can file a new issue.

**Comment by meganrogge** [comment]
Yes that is by design. Thanks for testing
